openedx_events/enterprise/data.py

```diff
@@ -94,3 +94,71 @@ class LedgerTransaction(BaseLedgerTransaction):
     parent_content_key = attr.ib(type=str, default=None)
     fulfillment_identifier = attr.ib(type=str, default=None)
     reversal = attr.ib(type=LedgerTransactionReversal, default=None)
+
+
+@attr.s(frozen=True)
+class EnterpriseCustomerUser:
+    """
+    Defines the common attributes of enterprise fulfillment classes.
```

Comment:
nit: looks like some copy/pasta in this docstring and `EnterpriseCourseEnrollment` below. Perhaps something like the following?

```suggestion
    Defines the attributes of enterprise customer users.
```

Reply:
thanks, good catch.